### PR TITLE
Snackbar list navigation

### PR DESCRIPTION
**Pull request by govindappaarun**
- add snack bar
- add list
- add navigation

### CODE REVIEW

File: src/components/input/index.css [hunk 1]
```diff
@@ -10,10 +10,29 @@ input with icon prefix
 .input-wrapper {
   display: flex;
   flex-direction: column;
-  margin: 0.5rem 0.5rem 1.2rem 0rem;
   position: relative;
   gap: 0.4rem;
 }
+
+.input-wrapper.w-icon {
```
Comment (vrx29) [inline src/components/input/index.css:17]: w-icon naming can be better. As its not easily understandable.

**Review by vrx29 — APPROVED** [review]
All other things look good.

**Review by Anurag021 — APPROVED** [review]
Code looks good to me.

**Comment by yash9263** [comment]
looks good to me

**Review by yash9263 — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -454,6 +460,140 @@ <h2 class="h2">Grid</h2>
...
+          <input type="text" class="input" />
+        </div>
+        <i class="far fa-heart"></i>
+        <i class="far fa-bell"></i>
+        <i class="fas fa-cart-plus"></i>
+        <div class="btn btn-link btn-link-secondary">Logout</div>
+      </div>
+    </div>
+
+    <div class="navbar my-1 navbar-dark">
```
Comment (hsnice16) [inline index.html:569]: As It is a navbar, you can use the `<nav>` semantic tag here, I think, it will help in more reachability.

**Review by hsnice16 — APPROVED** [review]
The code looks nice, just added a suggestion.

**Pull request merged**